src/partials/_html.hbs

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="de">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="initial-scale=1.0, width=device-width" />
+
+		<title>
+			Urlaubsverwaltung{{#if title}} - {{title}}{{/if}}
+		</title>
+
+		<link rel="stylesheet" href="/style.css" />
+		<link rel="preload" href="/page-prerenderer.js" as="script" />
+		{{#> pre}}{{/pre}}
+
+		<script>
+			if (window && window.history && window.history.scrollRestoration) {
+				window.history.scrollRestoration = "manual";
+			}
+		</script>
+	</head>
+
+	<body>
+		{{#> content}}{{/content}} {{#> scripts}}{{/scripts}}
+		<script defer src="/page-prerenderer.js"></script>
```

Comment:
die skripte wollen wir ggfs einfach als `type=module` laden. dann kommt uns da der IE nicht in die quere (und andere legacy browser)